WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/ScanStatusService.kt

```diff
@@ -72,8 +76,9 @@ class ScanStatusService
         return false
     }
 
-    private fun updateScanState(scanState: ScanStateModel?) {
-        _scanAvailable.value = scanState?.state != UNAVAILABLE && scanState?.state != UNKNOWN
+    private fun updateScanState(scanStateModel: ScanStateModel) {
+        _scanAvailable.value = scanStateModel.state != UNAVAILABLE && scanStateModel.state != UNKNOWN
+        _scanState.value = scanStateModel
     }
 
     private fun onScanStateFetched(scanStateError: ScanStateError?, isError: Boolean) {
```

Comment:
Minor (🔍 ): `isError` function parameter seems to be unused, is that needed for future?

Reply:
Yep, it will be needed in the future.